test/fixture.cpp

```diff
@@ -114,7 +141,7 @@ bool TestFixture::prepareTest(const char testname[])
 void TestFixture::teardownTest()
 {
     teardownTestInternal();
-
+    __gcov_flush();
```

Comment:
As this is done unconditionally it would interfere with other coverage executions. So this should be dependent on the environment you set up earlier.